pages/_app.js

```diff
@@ -15,11 +15,8 @@ function MyApp({ Component, pageProps }) {
     <header className="govuk-header " role="banner" data-module="govuk-header">
       <div className="govuk-header__container govuk-width-container">
         <div className="govuk-header__logo">
-          <a href="#" className="govuk-header__link govuk-header__link--homepage">
-            <span className="govuk-header__logotype">
-              <img src="/assets/images/govuk-logotype-crown.png" className="govuk-header__logotype-crown-fallback-image" width="36" height="32" alt="logo"/>
-            </span>
-          </a>
+          {/* eslint-disable-next-line @next/next/no-html-link-for-pages */}
```

Comment:
Please delete these comments if they are not required.

Reply:
it's required to top eslint from throwing up a linting error